config/deploy.rb

```diff
@@ -1,11 +1,16 @@
 # config valid only for current version of Capistrano
-lock '3.4.0'
+lock '3.3.5'
 
-set :application, 'my_app_name'
-set :repo_url, 'git@example.com:me/my_repo.git'
+set :application, 'aggressive-epsilon'
+set :repo_url, 'git@github.com:umts/aggressive-epsilon.git'
+set :user, 'rails'
```

Comment:
the user that deploys is whoever is deploying, im not sure what that user does (i think runs it)